### ISSUE TITLE
Source for mean_states.pth and var_states.pth

### ISSUE THREAD

**Issue by lxqpku** [likenneth/dialogue_action_token#3]
Where do the two files mean_states.pth and var_states.pth come from in the directory of redteaming_exp and how to achieve such two files for sotopia_env?

**Comment by likenneth** [comment]
They are basically the mean and variance of the states, for input normalization into the planner. 

To create such files for an environment, just extract all the last-token last-layer features for a set of inputs and call torch.mean() and torch.var().

**Comment by lxqpku** [comment]
> They are basically the mean and variance of the states, for input normalization into the planner.
> 
> To create such files for an environment, just extract all the last-token last-layer features for a set of inputs and call torch.mean() and torch.var().

Could you tell me what inputs are used for calculating the state_mean and state_std. Is the input the data for self-cloning or the data collected by td3 for RL training?

**Comment by likenneth** [comment]
I used the data for self-cloning.
